### ISSUE TITLE
Click here to see our new site not working

### ISSUE THREAD

**Issue by hitesh-opg** [OnePointGlobal/OnePoint-Global-Account-Toolkit#168]
Today when I click on Click here to see our new site it is not responding

Please see the recording (used debugger also so it helps )

[2017-06-06-1224-04.zip](https://github.com/OnePointGlobal/OnePoint-Global-Account-Toolkit/files/1054061/2017-06-06-1224-04.zip)

**Comment by deepika-Gopala** [comment]
Did  u logout of the site ? was the session previous days?

**Comment by hitesh-opg** [comment]
I was using a fresh session

now it seems to works though but strange behavior

**Comment by deepika-Gopala** [comment]
It would mostly be for the previous session. the passowrd and username , expires after 10 hours so its will be empty when the user tries to click on it and the link wouldnt work in that case,
 
We will discuss on this.

**Comment by hitesh-opg** [comment]
I am certain it was not 10 hour old session

yeah it will be good to know why such instances happen as they will have am impact on the user experience

**Comment by hitesh-opg** [comment]
This issue again does not happen frequently and will be good to check if we can find what could be causing such behavior for improving the user experience. 

It is not a critical but we should know if such instance happen with any client, what we can tell them so they get pass it.

**Comment by vivek-onepoint** [comment]
This issue has been fixed. You can have a look once we move the code to staging.

**Comment by hitesh-opg** [comment]
Hi Deepika,

I faced this issue again.

I was testing after deleting the browser history and for a couple of minutes the entire page was not responding. When I raised it last time, it was only the click here to see our new site.

 see recording
[2017-06-22-1459-47.zip](https://github.com/OnePointGlobal/OnePoint-Global-Account-Toolkit/files/1094393/2017-06-22-1459-47.zip)

I tried to remote connect to dev and staging server and I was able to log in, so the servers were online

After some time ( a minute or so) all back to normal.

As I said earlier, this is not a show stopper but will be good to find what is causing this behavior so it can be tackled and how we can make it better for the client to overcome it.
